assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -42,7 +42,17 @@
 		.googlesitekit-entity-search__close {
```

Comment:
@hussain-t Can we have the close button rounded as well? The button should be `36x36` on small screens and `32x32` on large screens to be consistent with the height of the input text.

Reply:
@hussain-t At `783px`, the search button is out of the input text. can you look into that?
![image](https://user-images.githubusercontent.com/31818/148047804-9b73c768-9b28-4b12-b428-706dee2c7e29.png)

Also, I forgot to mention, the button should be round on small screens only since it looks weird when it's round within the input text. When it's within the input text, let's make it square, to fill in the area within the input text.